controlplane/pkg/monitor_crossconnect_server/monitor_crossconnect_client.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
```

Comment:
Could we put this client code in the nsmd, as it looks like its quite specific to the nsmd.  The MonitorCrossConnectServer in this folder is quite intentionally *generic*, so it can be used by basically anyone who needs to use it :)

Reply:
Sure, will do as part of #506